src/main/java/org/duraspace/bagit/serialize/ZipBagSerializer.java

```diff
@@ -37,15 +35,13 @@ public Path serialize(final Path root) throws IOException {
             // it would be nice not to have to collect the files which are walked, but we're required to try/catch
             // inside of a lambda which isn't the prettiest. maybe a result could be returned which contains either a
             // Path or the Exception thrown... just an idea
-            final List<Path> files = Files.walk(root).collect(Collectors.toList());
+            final List<Path> files = Files.walk(root).toList();
```

Comment:
Same issue as `TarBagSerializer`, just need to move the Stream init to the try w/ resources.